src/styles/blocks/stopwatch.scss

```diff
@@ -0,0 +1,37 @@
+.stop-watch {
+  position: relative;
+  border: 2px dotted grey;
+  border-radius: 50%;
+  width: 2*$dial-radius;
+  height: 2*$dial-radius;
+
+  &__second-hand {
+    top: $dial-radius - $second-hand-length;
+    left: $dial-radius - $second-hand-width / 2;
+    @include hand(
+      second,
+      runSecondHand,
+      $second-hand-time,
+      $second-hand-angle,
+      $second-hand-steps
+    );
+    height: $second-hand-length;
+    width: $second-hand-width;
+    background-color: gray;
```

Comment:
Use a variable for the background-color of the second hand, making it easier to modify later.